eisbuk-admin/package.json

```diff
@@ -43,18 +43,21 @@
     "start": "react-app-rewired start",
     "start:eisbuk": "REACT_APP_EISBUK_SITE=eisbuk.web.app NODE_ENV=production react-app-rewired start --reset-cache",
     "start:igorice": "REACT_APP_EISBUK_SITE=igorice.web.app NODE_ENV=production react-app-rewired start --reset-cache",
-    "emulators:inspect": "npm --prefix \"./functions\" run build && exec firebase emulators:start --project eisbuk --inspect-functions 9229",
-    "emulators:start": "npm --prefix \"./functions\" run build && exec firebase emulators:start --project eisbuk",
+    "emulators:inspect": "yarn --cwd functions run build && exec firebase emulators:start --project eisbuk --inspect-functions 9229",
+    "emulators:start": "yarn --cwd functions run build && exec firebase emulators:start --project eisbuk",
     "build": "eslint . && react-app-rewired build",
-    "test": "npm --prefix \"./functions\" run build && firebase -c firebase-testing.json  emulators:exec --project eisbuk -- 'yarn run jest --watch --runInBand'",
-    "test:ci": "npm --prefix \"./functions\" run build && firebase -c firebase-testing.json  emulators:exec --project eisbuk -- 'yarn run jest --detectOpenHandles --ci --reporters=default --reporters=jest-junit --forceExit --runInBand'",
+    "test:emulators": "yarn --cwd functions build && firebase -c firebase-testing.json  emulators:exec --project eisbuk -- 'yarn jest --watch --runInBand src'",
+    "test:emulators:ci": "yarn --cwd functions build && firebase -c firebase-testing.json  emulators:exec --project eisbuk -- 'yarn jest --ci --reporters=default --reporters=jest-junit --collect-coverage --forceExit --runInBand'",
+    "test:quicktest": "echo Running tests with no emulators support && yarn jest --watch src",
+    "test": "echo 'Running all tests; using emulators.' && yarn test:emulators",
     "deploy:eisbuk": "firebase deploy --project eisbuk --only hosting:eisbuk",
     "deploy:production": "firebase deploy --project eisbuk --only hosting:igorice; firebase deploy --project eisbuk --except hosting",
     "deploy:preview": "firebase --project eisbuk hosting:channel:deploy",
     "eject": "react-scripts eject",
     "storybook": "start-storybook -p 6006 -s public",
     "build-storybook": "build-storybook -s public",
-    "chromatic": "npx chromatic --project-token dcmb4kmorxe"
+    "chromatic": "npx chromatic --project-token dcmb4kmorxe",
```

Comment:
It was actually bad practice to put this (now expired) project token here. Let's remove the `chromatic` script altogether! We won't be running it manyally.